dace/codegen/targets/cpu.py

```diff
@@ -410,17 +412,17 @@ def copy_memory(
         )
 
     def _emit_copy(
-        self,
-        sdfg,
-        state_id,
-        src_node,
-        src_storage,
-        dst_node,
-        dst_storage,
-        dst_schedule,
-        edge,
-        dfg,
-        stream,
+            self,
+            sdfg,
+            state_id,
+            src_node,
+            src_storage,
+            dst_node,
+            dst_storage,
+            dst_schedule,
+            edge,
+            dfg,
+            stream,
```

Comment:
Did you yapf this? I believe yapf makes 4 spaces, not 8.

Reply:
I didn't know about yapf before, but i just yapfed all files now..